patches/api/0175-Fix-Spigot-annotation-mistakes.patch

```diff
@@ -30,6 +30,20 @@ index 686235a2347ebeaa5654a14cdd717009f2c0105f..cf7f8a8f03adcbe466b59ea8b98b527f
      public static <T extends Keyed> Tag<T> getTag(@NotNull String registry, @NotNull NamespacedKey tag, @NotNull Class<T> clazz) {
          return server.getTag(registry, tag, clazz);
      }
+diff --git a/src/main/java/org/bukkit/GrassSpecies.java b/src/main/java/org/bukkit/GrassSpecies.java
+index f9c9ae463aacd593e3aa9caf037ea1e23d56c780..8147eb907320d48ccf731998063c1b41d706869b 100644
+--- a/src/main/java/org/bukkit/GrassSpecies.java
++++ b/src/main/java/org/bukkit/GrassSpecies.java
+@@ -6,7 +6,9 @@ import org.jetbrains.annotations.Nullable;
+ 
+ /**
+  * Represents the different types of grass.
++ * @deprecated use BlockData
```

Comment:
paper comment

Reply:
actually, can you have comments here? since it is a comment? guess its fine

Reply:
> actually, can you have comments here? since it is a comment?

not without breaking the javadoc (or having it appear)